src/entities/shortNames.js

```diff
@@ -33,30 +33,29 @@ function getDisplayName(name) {
 }
 
 function removeCorpNames(name) {
-    let uname = name.replace(/\+|\/|\&|\.|-|\+| /g, '')
+    const uname = name.replace(/\+|\/|&|\.|-|\+| /g, '')
     let shortName = name
     for (const index in sortedCorpList) {
-        let suffix = new RegExp(sortedCorpList[index] + '$', 'i')
-        let prefix = new RegExp('^' + sortedCorpList[index], 'i')
+        const suffix = new RegExp(sortedCorpList[index] + '$', 'i')
+        const prefix = new RegExp('^' + sortedCorpList[index], 'i')
         if (uname.match(suffix)) {
-            let matchingSuffix = uname.match(suffix)[0]
+            const matchingSuffix = uname.match(suffix)[0]
             // match on suffixes that use spaces or other chars between letters
             // max seems to be 4 chars: m.b.H. & Co KG
-            let matchingSpacedSuffix = matchingSuffix.split('').join('\\W{0,4}')
+            const matchingSpacedSuffix = matchingSuffix.split('').join('\\W{0,4}')
             // add non-word char to regex so inc doesn't match Zinc
-            let spacedRegex = new RegExp('\\W' + matchingSpacedSuffix + '$', 'i')
+            const spacedRegex = new RegExp('\\W' + matchingSpacedSuffix + '$', 'i')
             // remove trailing punctuation, in order to check for match
-            let matchName = name.replace(/(\.|,)$/, '')
+            const matchName = name.replace(/(\.|,)$/, '')
 
             if (matchName.match(spacedRegex)) {
                 shortName = matchName.replace(spacedRegex, '')
                 break
             }
         } else if (uname.match(prefix)) {
-            let matchingPrefix = uname.match(prefix)[0]
-            let matchingSpacedPrefix = matchingPrefix.split('').join('\\W{0,1}')
-            let spacedRegex = new RegExp('^' + matchingSpacedPrefix + '\\W{0,1}\\s', 'i')
-            let matchName = name.replace(/^\./, '')
```

Comment:
This wasn't used - not sure if bug or not? cc @dharb

Reply:
Ah yeah looks like a remnant of old code that I revised.